spec/features/managing_users_spec.rb

```diff
@@ -66,6 +66,22 @@
       expect(page).to have_text "admin@example.com" # ...with user data
     end
 
+    scenario "I see user pledging history on the user page" do
+      user    = create(:user)
+      item    = create(:item)
+      pledge  = user.pledges.create!(item_id: item.id)
```

Comment:
The tests are blowing up here too, right? Pledges are actually associated with `wishlist_items`, not `items` themselves (since we need to know where to send the donation). Changing to a wishlist_item should get it past this point.